Gemfile

```diff
@@ -34,6 +34,7 @@ gem 'jbuilder', '~> 2.5'
 # gem 'bcrypt', '~> 3.1.7'
 gem 'paperclip', '~> 5.0.0'
 gem 'aws-sdk', '~> 2.3'
```

Comment:
You can remove the aws-sdk gem since we wont be dealing with it anymore @CoderRx001